gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/TraversalInterruptionComputerTest.java

```diff
@@ -86,7 +97,7 @@ public void shouldRespectThreadInterruptionInVertexStep() throws Exception {
 
         // total time for test should not exceed 5 seconds - this prevents the test from just hanging and allows
         // it to finish with failure
-        assertThat(startedIterating.await(5000, TimeUnit.MILLISECONDS), CoreMatchers.is(true));
+        assertThat(startedIterating.await(1000, TimeUnit.MILLISECONDS), CoreMatchers.is(true));
```

Comment:
Why not just leave this as 5s?  Seems to work against the fix.  What is the actual timeout time value?